src/actions/classifier.js

```diff
@@ -25,6 +25,11 @@ export function startNewClassification(workflowID) {
     }).then(() => {
       return dispatch(setNeedsTutorial())
     }).then(() => {
+      if (getState().classifier.guide[workflowID] !== undefined) {
+        return
+      }
+      return dispatch(fetchFieldGuide(workflowID))
+    }).then(() => {
```

Comment:
Well, it turns out there isn't much that doesn't have a dependent, just the Field Guide and setNeedsTutorial (which actually needs to be loaded before the screen loads so that it goes straight to the tutorial rather than opening the classifier and then moving to the tutorial) so it didn't seem to buy much.